app/models/article.rb

```diff
@@ -22,4 +22,5 @@ def owned_by?(owner)
     return false unless owner.is_a?(User)
     user == owner
   end
+
```

Comment:
nit: can remove this new line

Reply:
removed duplicate line

Reply:
I still see it in here, have you committed the change to remove it?

Reply:
removed new line